dev/TeachingTip/TeachingTip.h

```diff
@@ -211,16 +213,16 @@ class TeachingTip :
     static constexpr float s_defaultTipHeightAndWidth = 320;
 
     //Ideally this would be computed from layout but it is difficult to do.
-    static constexpr float s_beakOcclusionAmount = 2;
-
-    // The beak is designed as an 8x16 pixel shape, however it is actual a 10x20 shape which is partially occluded by the tip content.
-    // This is done to get the border of the tip to follow the beak shape without drawing the border on the tip edge of the beak.
-    inline float MinimumTipEdgeToBeakEdgeMargin() { return static_cast<float>(m_beakOcclusionGrid.get().ColumnDefinitions().GetAt(1).ActualWidth() + s_beakOcclusionAmount); };
-    inline float MinimumTipEdgeToBeakCenter() { return static_cast<float>(m_beakOcclusionGrid.get().ColumnDefinitions().GetAt(0).ActualWidth() +
-                                                m_beakOcclusionGrid.get().ColumnDefinitions().GetAt(1).ActualWidth() +
-                                                (std::max(m_beakPolygon.get().ActualHeight(), m_beakPolygon.get().ActualWidth()) / 2)); }
-
-    inline float BeakLongSideActualLength() { return static_cast<float>(std::max(m_beakPolygon.get().ActualHeight(), m_beakPolygon.get().ActualWidth())); }
-    inline float BeakLongSideLength() { return static_cast<float>(BeakLongSideActualLength() - (2 * s_beakOcclusionAmount)); }
-    inline float BeakShortSideLength() { return static_cast<float>(std::min(m_beakPolygon.get().ActualHeight(), m_beakPolygon.get().ActualWidth()) - s_beakOcclusionAmount); }
+    static constexpr float s_pointerOcclusionAmount = 2;
+
+    // The Pointer is designed as an 8x16 pixel shape, however it is actual a 10x20 shape which is partially occluded by the tip content.
```

Comment:
Pointer vs. pointer. It was 'beak'. Did you intend to capitalize in all comments?

Reply:
it is actual a 10x20 ==> it is actually a 10x20